LondonBritishLibrary/add/BLadd19339.xml

```diff
@@ -0,0 +1,189 @@
+                        ወበመንግሥቱ፡ <persName ref="PRS3692egwalas">ለንጉሥነ፡ እጓለ፡ ጽዮን፡</persName> በቅድመ፡
+                        ኵሎሙ፡ ሊቃውንት፡ እለ፡ ተሰዱ፡ በኢትዮጵያ፤ በእንተ፡ ዛቲ፡
+                        ርትዕት፡ ሃይማኖተ፡ እስክንድያ፤ መንበረ፡ ስብከቱ፡ ለማርቆስ፡ ሐዋርያ።<gap reason="ellipsis"></gap>
+                        </colophon>
+                            </msItem>
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">21</measure>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>8.75</height>
+                                        <width>6.125</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>One unnumbered blank leaf at the beginning.</foliation>
+                                <condition key="good">Stained by water.</condition>
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18 19">
+                                </layout>
+
+
+                            </layoutDesc>
+
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neatly written.</desc>
+                                <date notBefore="1808" notAfter="1867"/>
+                            </handNote>
+
+                        </handDesc>
+
+
+                        <additions>
+                            <list>
+
+                                <item xml:id="a1">
+                                    <desc type="Unclear">Comment on the manuscript content in
+                                      English.
+                                    </desc>
+                                    <q xml:lang="en">Copy of the Patriarch of Alexandria's
+                                    Instructions to the Abyssinians on points of Religion, much
+                                  esteemed by the Abyssinians and the Ras, and presented to
+                                  <persName ref="PRS8442Salt">H. Salt</persName>
+                                by him.</q>
+
+                                </item>
+
+
+                            </list>
+                        </additions>
+
+
+                    </physDesc>
+
+                    <history>
+                        <origin>
+                            <origDate notBefore="1808" notAfter="1867" evidence="lettering"/>
```

Comment:
the ante quem could probably be the date of Salt's return from Ethiopia? (That's how I would understand the English note, or do you think the letter was copied from the one given to Salt?)